modules/op-chains-signing-secrets-skopeo.adoc

```diff
@@ -4,7 +4,7 @@
 :_mod-docs-content-type: PROCEDURE
 
 [id="chains-signing-secrets-skopeo_{context}"]
-= Signing using skopeo
+= Generating the cosign key pair with the skopeo tool
```

Comment:
I see no reason to mention `cosign` here

Reply:
@ppitonak 
If we now have a chapter about generating the x509 key pair, I thought it would be clearer to mention here that the chapter is to use for the  the cosign signing scheme.

However, maybe the original titles made more sense : Signing with the skopeo tool, Signing with the cosign tool

And we could call the new x509 chapter "Signing by using TektonConfig CR" , WDYT?

Reply:
WDYT @PuneetPunamiya @jkhelil ?

Reply:
No I think the first heading was correct and I agree with @ppitonak

Reply:
@PuneetPunamiya  Yes sounds good, thanks! It is just still not clear to me what to call the new section title. Should it be the current "Generating the x509 key pair" or rather something like "Signing by using TektonConfig CR" (similarly to the scopeo/cosign titles)?

Reply:
If you see the further section it says all about how using `skopeo` to generate the signing keys
So the title `Signing using Skopeo` should be enough

Reply:
Hello @PuneetPunamiya I am sorry I think we misunderstood each other here. I understand about the Skopeo heading, what I am asking now is the heading to the new section for `x509`, not the skopeo one: [Generating the x509 key pair](https://84253--ocpdocs-pr.netlify.app/openshift-pipelines/latest/secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.html#chains-signing-secrets-ecdsa_using-tekton-chains-for-openshift-pipelines-supply-chain-security)
If that title for `x509` makes sence, or if to would be better to call it something like "Signing by using TektonConfig CR"
I hope my question is clear now :pray: Thank you!